src/hipscat_import/resume_files.py

```diff
@@ -0,0 +1,96 @@
+"""Methods for reading and writing intermediate files for import execution."""
+
+import os
+import shutil
+from datetime import datetime
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from hipscat import pixel_math
+
+
+def read_histogram(tmp_path, highest_healpix_order):
+    """Read a numpy array at the indicated directory.
+    Otherwise, return histogram of appropriate shape."""
+    if os.path.exists(os.path.join(tmp_path, "mapping_histogram.csv")):
+        return np.loadtxt(
+            os.path.join(tmp_path, "mapping_histogram.csv"),
+            dtype=np.ulonglong,
+            delimiter=",",
+        )
```

Comment:
Could we use a binary file instead of text for this? Performance and storage might be better. [Some options compared here.](https://stackoverflow.com/questions/9619199/best-way-to-preserve-numpy-arrays-on-disk)

Reply:
Yup!

Reply:
Awesome, thanks!